app/src/main/java/zapsolutions/zap/fragments/SendBSDFragment.java

```diff
@@ -616,10 +624,6 @@ public void onClick(View v) {
             }
         });
 
-
-        // Calculate fee
-        calculateFee();
```

Comment:
I think we don't need to calculate the fee at this place.
The change listener is already in place that triggers the fee calculation 
here: https://github.com/LN-Zap/zap-android/pull/38/files#diff-8118f2a5a6d859493c94f5aa7debddd9L306 
when we set an received amount 
here: https://github.com/LN-Zap/zap-android/pull/38/files#diff-8118f2a5a6d859493c94f5aa7debddd9L465

Otherwise the user has to type the amount and the fee calculation will be triggered again.